react/features/base/lib-jitsi-meet/_.web.js

```diff
@@ -1,3 +1,8 @@
-/* global JitsiMeetJS */
+/* @flow */
+
+// Polyfill for URL constructor
+import 'url-polyfill';
```

Comment:
IE, It doesn't support URL API. I mentioned it in the PR description (about redirections in IE)